internal/pkg/cli/cli.go

```diff
@@ -20,56 +20,51 @@ import (
 
 // GlobalOpts holds fields that are used across multiple commands.
 type GlobalOpts struct {
-	projectName string
-	prompt      prompter
+	appName string
+	prompt  prompter
 }
 
-// NewGlobalOpts returns a GlobalOpts with the project name retrieved from viper.
+// NewGlobalOpts returns a GlobalOpts with the application name retrieved from viper.
 func NewGlobalOpts() *GlobalOpts {
-	bindProjectName()
+	bindAppName()
 
 	return &GlobalOpts{
-		// Leave the projectName as empty in case it's overwritten by a global flag.
+		// Leave the appName as empty in case it's overwritten by a global flag.
 		// See https://github.com/aws/amazon-ecs-cli-v2/issues/570#issuecomment-569133741
 		prompt: prompt.New(),
 	}
 }
 
-// ProjectName returns the project name.
+// AppName returns the application name.
 // If the name is empty, it caches it after querying viper.
-func (o *GlobalOpts) ProjectName() string {
-	if o.projectName != "" {
-		return o.projectName
+func (o *GlobalOpts) AppName() string {
+	if o.appName != "" {
+		return o.appName
 	}
-	o.projectName = viper.GetString(projectFlag)
-	return o.projectName
+	o.appName = viper.GetString(projectFlag)
```

Comment:
Should this flag be renamed as `appFlag`? Or leave it to the other PRs up to you.

Reply:
Done